src/main/resources/config.yml

```diff
@@ -78,4 +78,7 @@ EnableAntiRadar: false # Enable invisibility for craft pilots to nullify minimap
 
 # Fire Behavior
 EnableFireballPenetration: false # Randomly replace blocks with fire when they are struck by fireballs.
-AddFiresToHitbox: true # Add fires lit on the surface of a craft to its hitbox so they don't obstruct it.
\ No newline at end of file
+AddFiresToHitbox: true # Add fires lit on the surface of a craft to its hitbox so they don't obstruct it.
+
+# Piston Behavior
+ReImplementTranslocation: false
```

Comment:
This should have a comment to describe what it does and that it only works on 1.11+.